packages/runtime/container-runtime/package.json

```diff
@@ -74,7 +74,6 @@
 		"@fluidframework/driver-definitions": "workspace:~",
 		"@fluidframework/driver-utils": "workspace:~",
 		"@fluidframework/garbage-collector": "workspace:~",
-		"@fluidframework/protocol-base": "^0.1039.1000",
```

Comment:
this is awesome! container runtime will no longer have direct dependencies on server!